privacyidea/cli/privacyideatokenjanitor/findtokens.py

```diff
@@ -87,46 +86,38 @@
 * delete
 
 
-    privacyidea-token-janitor find 
+    privacyidea-token-janitor find
         --last_auth=10h|7d|2y
-        --tokeninfo-key=<key>
+        --tokeninfo= 'token_info_key == token_info_value' or 'token_info_key >= token_info_value'
+                        or 'token_info_key <= token_info_value'
         --has-not-tokeninfo-key<key>
         --has-tokeninfo-key<key>
-        --tokeninfo-value=<value>
-        --tokeninfo-value-greater-than=<value>
-        --tokeninfo-value-less-than=<value>
-        --tokeninfo-value-after=<value>
-        --tokeninfo-value-before=<value>
         --assigned false
         --orphaned true
         --tokentype=<type>
         --serial=<regexp>
         --description=<regexp>
-        
-        --action={0!s}
-        
-        --set-description="new description"
-        --set-tokeninfo-key=<key>
-        --set-tokeninfo-value=<value>    
-    
+
+
 .. note:: If you fail to redirect the output of this command at the commandline
    to e.g. a file with a UnicodeEncodeError, you need to set the environment
    variable like this:
        export PYTHONIOENCODING=UTF-8
    Here is a good read about it:
    https://stackoverflow.com/questions/4545661/unicodedecodeerror-when-redirecting-to-file
 
-""".format("|".join(ALLOWED_ACTIONS))
+"""
+
+ALLOWED_ACTIONS = ["disable", "delete", "unassign", "mark", "export", "listuser", "tokenrealms"]
 
 app = create_app(config_name='production', silent=True)
```

Comment:
we don't need to create the `app` here. This should be done by the FlaskGroup automatically.